### PR TITLE
🧹 remove todo comments from docs

### LINKED ISSUE #598: Exclude comments that start in TODO from the docs generation
**Describe the bug**
When running `make lr/docs/markdown` we're generating docs from TODO comments in resources. We should be skipping lines that start with TODO.

**To Reproduce**
1. Run `make lr/docs/markdown`
2. Check the docs generated for the gcp-pack

**Expected behavior**
No internal notes make it to the docs

### PR DESCRIPTION
**Pull request by jaym**
Additionally:
- 🐛 Fix field comments in gcp.lr : see commit messages

Fixes #598

### CODE REVIEW

File: resources/packs/gcp/gcp.lr [hunk 1]
```diff
@@ -581,7 +581,8 @@ private gcp.sql.instance @defaults("name") {
   // Instance state
   state string
   // Project
-  project string // TODO: should this be id or name
+  // TODO: should this be id or name
```
Comment (tas50) [inline resources/packs/gcp/gcp.lr:588]: This one is still showing up in the docs with this change

Comment (jaym) [reply resources/packs/gcp/gcp.lr:588]: hmm, maybe that uses a way that is different than the shell. let me look into it

Comment (jaym) [reply resources/packs/gcp/gcp.lr:588]: should be fixed now

Comment (tas50) [reply resources/packs/gcp/gcp.lr:588]: It's getting closer, but now the "Project" description is getting removed as well

Comment (tas50) [reply resources/packs/gcp/gcp.lr:588]: Working now

**Review by vjeffrey — APPROVED** [review]

**Pull request merged**